packages/mds-agency/request-handlers.ts

```diff
@@ -149,8 +147,7 @@ export const getVehiclesByProvider = async (req: AgencyApiRequest, res: AgencyGe
 
   try {
     const response = await getVehicles(skip, take, url, req.query, provider_id)
-    const { version } = res.locals
-    return res.status(200).send({ version, ...response })
+    return res.status(200).send({ ...response })
```

Comment:
Still not sure the new object syntax is required in all of these places `{ ...response }` and `response` being equivalent and all.

Reply:
It's the difference between `{ response: {...}}` and `{	"vehicles": [ ... ]
 	"links": {
        "first": "https://...",
        "last": "https://...",
        "prev": "https://...",
        "next": "https://..."
    }}`, and the former is not in the spec.

Reply:
There is _no_ difference between `.send({...response})` and `.send(response)` ... note that I didn't suggest `.send({ response })`

Reply:
I agree with Michael, `.send({ ...response })` and `.send(response)` are the same AFAIK

Reply:
never mind. never mind.